htdocs/install/mysql/tables/llx_element_tag.key.sql

```diff
@@ -1,21 +0,0 @@
--- ============================================================================
```

Comment:
It seems this file is removed instead of being renamed like it is for llx_element_tag.

Reply:
in fact i forgot to create llx_element_categorie.key.sql, it's done